app/views/topics/edit.html.erb

```diff
@@ -0,0 +1,2 @@
+<h1>Topics#edit</h1>
```

Comment:
Please refactor this into a form partial so that you can use it for both the Edit and New view.